src/aap_eda/settings/default.py

```diff
@@ -204,6 +205,10 @@ def _get_boolean(name: str, default=False) -> bool:
 # See https://cfpb.github.io/django-flags/
 FLAGS = {}
 
+for feature in list(FLAGS.keys()):
+    if feature in settings:
+        FLAGS[feature][0]["value"] = settings.get(feature)
```

Comment:
Are we not going to support multiple conditions? https://cfpb.github.io/django-flags/usage/#defining-flags

Reply:
In this first iteration, we will only support booleans